azure-samples/pom.xml

```diff
@@ -87,6 +87,16 @@
             <groupId>com.azure</groupId>
             <artifactId>azure-storage-blob</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.microsoft.azure</groupId>
+            <artifactId>azure-documentdb</artifactId>
+            <version>1.9.4</version>
+        </dependency>
+        <dependency>
+            <groupId>joda-time</groupId>
+            <artifactId>joda-time</artifactId>
```

Comment:
I don't think we should do this explicitly in track2 (it is ok if it from another lib, but then you will not need to write it here). Check your code and make refactor.

Reply:
done